src/fe-small-advertisements/.eslintrc.js

```diff
@@ -1,3 +1,14 @@
 module.exports = {
-  extends: ["eslint-config-small-ads"],
+  extends: ['eslint-config-small-ads'],
+  rules: {
+    'import/prefer-default-export': 'off',
```

Comment:
One thing, and this isnt something you could know of. But think of this situation like this. We want to have this rule for all our projects. Meaning exporting as constancts and not as default. Because of that we want this to be in the global `eslint` package, otherwise we will be repeating it each time. Now the one for the `ui` is unique to it and this is why we arent including it in the global eslint config. 

Do I explain that well?

Reply:
Yes, you did. Very well!

 I already got it when you said:

> Hey, we want this one to be global but the "import/no-extraneous-dependencies": "off", is specific to the ui.

That is why I said:

> Oh, I saw I had to use it in both projects, but I wasn't not sure if we would need it in every project in case we create something new. But it makes sense that we have it global for now if we are using it in every project. Total sense.

I pushed what I already committed yesterday, that is why this issue wasn't fixed yet. I didn't want to fix it in the previous commit to not make a mess, is that wrong? I mean, to push what I already committed before making more changes even though you made comments I could fix

Reply:
Anyhow, your explanation now was very clear and it draws a complete picture of the issue.